bundle/ScheduledVisibility/Handler/ContentAndLocation.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Handler;
+
+use Ibexa\Contracts\Core\Repository\ContentService;
+use Ibexa\Contracts\Core\Repository\LocationService;
+use Ibexa\Contracts\Core\Repository\Repository;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\ScheduledVisibilityInterface;
+
+final class ContentAndLocation implements ScheduledVisibilityInterface
+{
+    public function __construct(
+        private readonly Repository $repository,
+        private readonly ContentService $contentService,
+        private readonly LocationService $locationService,
+    ) {}
+
+    public function hide(Content $content): void
+    {
+        $this->repository->sudo(
+            function () use ($content): void {
+                $this->contentService->hideContent($content->getContentInfo());
```

Comment:
Should we inject the `Content`  and `Location` handlers here and call those instead of duplicating the implementation?

Reply:
Not worth it from my POV, the implementation seems too small.

Reply:
fixed in 2749908b96c098da8b38f8fc38193587de872769